source/common/router/config_impl.h

```diff
@@ -288,6 +292,24 @@ class HashPolicyImpl : public HashPolicy {
   std::vector<HashMethodPtr> hash_impls_;
 };
 
+/**
+ * Implementation of HedgePolicy that reads from the proto route or virtual host config.
+ */
+class HedgePolicyImpl : public HedgePolicy {
+
+public:
+  HedgePolicyImpl(const envoy::api::v2::route::HedgePolicy& hedge_policy);
```

Comment:
make this `explicit` to avoid accidental implicit conversion from a `HedgePolicy`